src/foam/u2/wizard/ScrollWizardletView.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard',
+  name: 'ScrollWizardletView',
+  extends: 'foam.u2.wizard.WizardletView',
+
+  documentation: `Simply displays wizardlets consecutively.`,
+
+  imports: [
+    'notify',
+    'stack'
+  ],
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  messages: [
+    { name: 'ACTION_LABEL', message: 'Submit' },
+    { name: 'SAVE_IN_PROGRESS', message: 'Saving...' },
+    { name: 'ERROR_MSG', message: 'Information was not successfully submitted, please try again later' },
+    { name: 'ERROR_MSG_DRAFT', message: 'An error occured while saving your progress.' },
+    { name: 'SUCCESS_MSG', message: 'Information successfully submitted.' },
+    { name: 'SUCCESS_MSG_DRAFT', message: 'Your progress has been saved.' },
+  ],
+
+  css: `
+    ^ {
+      margin: 30px;
+    }
+  `,
+
+  requires: [
+    'foam.u2.detail.VerticalDetailView'
+  ],
+
+  properties: [
+    {
+      class: 'Boolean',
+      name: 'isErrorFree',
+      expression: function(wizardlets) {
+        var check = true;
+        wizardlets.forEach(wizardlet => {
+          if ( ! wizardlet.readyToSubmit() ) {
+            check = false;
+          }
+        });
+        return check;
+      }
+    }
+  ],
+
+  methods: [
+    function initE() {
+      var self = this;
+      this.addClass(this.myClass());
+      this.start('h1').add(this.title).end()
+        .start()
+        .add(this.slot(
+          wizardlets => {
+            return this.E().forEach(
+              wizardlets.filter(section => section.of),
+              wizardlet => {
+                var subThis = this.startContext({});
+                subThis.__subSubContext__.register(
+                  this.VerticalDetailView,
+                  'foam.u2.detail.SectionedDetailView'
+                );
```

Comment:
This replaces SectionedDetailView with VerticalDetailView to make the wizard flat. We don't want cards displaying in the wizard because we want to be able to compose multiple FObjectProperty in a section for a seamless wizard screen.

Reply:
So just to expand Eric's comment - which I agree with - is the default of FObjectProperties is SectionedDetailView, which gives a white card background. We want to override all FObjectProperties to use a standard.